### PR TITLE
fix(zero): Made zero health endpoint to return json as alpha's health

### LINKED ISSUE #5517: Zero's /health endpoint returns just "Ok" where the same Alpha's endpoint returns a JSON
I'm testing Dgraph 1.2.3 version and notice there is a difference in `/health` endpoints for Zero and Alpha. It would be good to unify that endpoint to return the same JSON structure.

Zero's response to `curl http://localhost:6080/health`
```
Ok
```

Alpha's reponse to `curl http://localhost:8080/health`:
```
{"version":"v1.2.3","instance":"alpha","uptime":59131}
```

### PR DESCRIPTION
**Pull request by tharun208**
Signed-off-by: Tharun <rajendrantharun@live.com>

fix(zero): Made zero health endpoint to return JSON as alpha's health

Fixes #5517

---
This change is [Reviewable](https://reviewable.io/reviews/dgraph-io/dgraph/5723)

### CODE REVIEW

File: dgraph/cmd/zero/http.go [hunk 1]
```diff
@@ -235,8 +237,27 @@ func (st *state) getState(w http.ResponseWriter, r *http.Request) {
 func (st *state) pingResponse(w http.ResponseWriter, r *http.Request) {
 	x.AddCorsHeaders(w)
 
+	var healthAll []pb.HealthInfo
```
Comment (parasssh) [inline dgraph/cmd/zero/http.go:240]: Why is this a slice? It will just have 1 item.

File: dgraph/cmd/zero/http.go [hunk 2]
```diff
@@ -235,8 +237,27 @@ func (st *state) getState(w http.ResponseWriter, r *http.Request) {
...
+
+	// Append Self
+	healthAll = append(healthAll, pb.HealthInfo{
+		Instance: "zero",
+		Address:  x.WorkerConfig.MyAddr,
+		Status:   "healthy",
+		Version:  x.Version(),
+		Uptime:   int64(time.Since(x.WorkerConfig.StartTime) / time.Second),
+		LastEcho: time.Now().Unix(),
```
Comment (parasssh) [inline dgraph/cmd/zero/http.go:249]: Also set Group: "0"

File: dgraph/cmd/zero/http.go (hunk 1 above)
Comment (tharun208) [reply dgraph/cmd/zero/http.go:240]: to make it consistent with zero's health response structure

**Comment by tharun208** [comment]
> Could you also post the output with your changes?

Screen Shot 2020-06-26 at 12 48 51 PM

**Comment by manishrjain** [comment]
FYI, @martinmr

**Comment by tharun208** [comment]
can we merge this ?. I am not a member and I cant merge this by default

**Review by tharun208 — COMMENTED** [review]
> Reviewed 1 of 1 files at r2.
> *[Reviewable](https://reviewable.io/reviews/dgraph-io/dgraph/5723)* status: all files reviewed, 2 unresolved discussions (waiting on @manishrjain, @parasssh, and @vvbalaji-dgraph)

**Comment by tharun208** [comment]
coming after 1 year. can we merge this  😅 ?

**Comment by tharun208** [comment]
@parasssh is it good to merge or can i close this ?

**Comment by MichelDiz** [comment]
hey @tharun208, sorry for all this delay. We have renamed master to main. So, you need to re-do the PR pointing to main. Or maybe you could rebase this PR. I can point it to main from my side and then you fix it doing rebase. We can try that.

Or just close it and reopen. Your call.

**Comment by MichelDiz** [comment]
PS. I gonna track this for a month. If you don't see that, I'll try to do something myself. If you can't keep your name in the commits, the solution will be to copy the code and do the PR myself.

**Comment by mangalaman93** [comment]
closing this PR in lieu of #8766

**Pull request closed without merging**